DataFormats/Detectors/Common/include/DetectorsCommonDataFormats/DetID.h

```diff
@@ -109,13 +110,14 @@ class DetID
 
   static constexpr const char* sDetNames[nDetectors + 1] = ///< defined detector names
 
-    {"ITS", "TPC", "TRD", "TOF", "PHS", "CPV", "EMC", "HMP", "MFT", "MCH", "MID", "ZDC", "FT0", "FV0", "FDD", "ACO", nullptr};
+    {"ITS", "TPC", "TRD", "TOF", "PHS", "CPV", "EMC", "HMP", "MFT", "MCH", "MID", "ZDC", "FT0", "FV0", "FDD", "ACO", "ITS3", nullptr};
```

Comment:
I am wondering if we will run into problems with DPL headers. I remember that detector origins origins can only be 3 letter, or maybe I am wrong?

Reply:
@sawenzel true, better have it as 3 chars, e.g. IT3.
Strictly speaking, the mapping from the DetID to getectorOrigin is done in the yet unmerged https://github.com/AliceO2Group/AliceO2/pull/3685: https://github.com/AliceO2Group/AliceO2/blob/c242bda0dc1367017b1491bb5f3a21cf14eb1575/DataFormats/Detectors/Common/include/DetectorsCommonDataFormats/DetID.h#L138-L141, and since we will never have 
ITS and ITS3 at the same time, we could use the same gDataOriginITS for both, but in many places in the code we use the name instead of the origin, so better to have them the same.

Reply:
> I am wondering if we will run into problems with DPL headers. I remember that detector origins origins can only be 3 letter, or maybe I am wrong?

Yes, I was also wondering, I'll make it `IT3`.

Reply:
@shahor02 @mconcas Just to make sure: We will then consider ITS3 to comprise both the new IB and the old OB, correct? So, ITS3 will have to duplicate the OB-related geometry and code?

Reply:
Yes, it can be a struggle in case of modifications on the OB of the ITS, but at the same time minimises the probability to break something in production code.

Reply:
@qgp yes, I would completely duplicate all layers anyway.